DAML-LF: Kill RelativeContractId (#5991)
digital-asset/daml: daml-lf/spec/contract-id.rst

  discriminator is not `fresh <Discriminator Freshness_>`_, abort the
  interpretation. The submitter can restart the interpretation, which will pick
  another submission seed.
* If the transaction succeeds, the output is a *submitted transaction*

> Depending of the ledger implementation, the local contract IDs are

Reviewer: ```suggestion
Depending on the ledger implementation, the local contract IDs are
```

> suffixed with a suffix in a latter step. This yields the *committed

Reviewer: ```suggestion
suffixed with a suffix in a later step. This yields the *committed
```

transaction*. For ledgers that do not require suffixing, committed and submitted

> transaction coincide. Committed transactions are the source of true to

Reviewer: ```suggestion
transactions coincide. Committed transactions are the source of truth to
```

> describe the state if the ledger.

Reviewer: ```suggestion
derive the state of the ledger.
```
